### PR TITLE
(minor/OCD) SQL: Start IDs at 1

### PR DESCRIPTION
**Pull request by jnccloud**
Very, very minor change to the SQL file to start the records at 1 instead of these random values.

### CODE REVIEW

File: qb-clothing.sql [hunk 1]
```diff
@@ -7,7 +7,7 @@ CREATE TABLE IF NOT EXISTS `playerskins` (
   PRIMARY KEY (`id`),
   KEY `citizenid` (`citizenid`),
   KEY `active` (`active`)
-) ENGINE=InnoDB AUTO_INCREMENT=43010 DEFAULT CHARSET=latin1;
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=latin1;
```
Comment (Z3rio) [inline qb-clothing.sql:10]:
```suggestion
) ENGINE=InnoDB AUTO_INCREMENT DEFAULT CHARSET=latin1;
```

File: qb-clothing.sql [hunk 2]
```diff
@@ -19,4 +19,4 @@ CREATE TABLE IF NOT EXISTS `player_outfits` (
   PRIMARY KEY (`id`),
   KEY `citizenid` (`citizenid`),
   KEY `outfitId` (`outfitId`)
-) ENGINE=InnoDB AUTO_INCREMENT=8970 DEFAULT CHARSET=utf8mb4;
\ No newline at end of file
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4;
```
Comment (Z3rio) [inline qb-clothing.sql:22]:
```suggestion
) ENGINE=InnoDB AUTO_INCREMENT DEFAULT CHARSET=utf8mb4;
```

**Review by Z3rio — CHANGES_REQUESTED** [review]
The default value is already 1, no need to have `=1`

**Comment by jnccloud** [comment]
> The default value is already 1, no need to have `=1`

Resolved

**Review by Z3rio — APPROVED** [review]
LGTM

**Pull request merged**